code/modules/mining/ores_coins.dm

```diff
@@ -471,9 +509,10 @@ GLOBAL_LIST_INIT(sand_recipes, list(\
 						"uranium" = /obj/item/clothing/gloves/ring/uranium)
 	var/typekey = typelist[cmineral]
 	if(ispath(typekey))
-		to_chat(user, "<span class='notice'>You make [src] into a ring.</span>")
-		new typekey(get_turf(loc))
+		to_chat(user, "<span class='notice'>You carefully cut a hole into [src] turning it into a ring.</span>")
+		var/obj/item/clothing/gloves/ring/ring = new typekey()
 		qdel(src)
+		user.put_in_hands(ring)
```

Comment:
```suggestion
		new typekey(get_turf(loc))
		qdel(src)
```
My bad, this could cause stuff like cutting stuff on the ground, which would then magically be put in your hands

Reply:
Said on discord but i'll mirror it here too in case anyone else wants to bring up anything.
I personally don't view this as an issue, cutting the coin will (icly) involve picking it up, so i wouldn't quite call it magical.